packages/celotool/src/cmds/account/faucet.ts

```diff
@@ -40,18 +53,26 @@ export const handler = async (argv: FaucetArgv) => {
     console.log(`Using account: ${account}`)
     kit.defaultAccount = account
 
-    const [goldToken, stableToken] = await Promise.all([
+    const [goldToken, stableToken, reserve] = await Promise.all([
       kit.contracts.getGoldToken(),
       kit.contracts.getStableToken(),
+      kit.contracts.getReserve(),
     ])
-    const goldAmount = (await convertToContractDecimals(1, goldToken)).toString()
-    const stableTokenAmount = (await convertToContractDecimals(10, stableToken)).toString()
-
-    console.log(`Fauceting ${goldAmount} Gold and ${stableTokenAmount} StableToken to ${address}`)
-    await Promise.all([
-      goldToken.transfer(address, goldAmount).sendAndWaitForReceipt(),
-      stableToken.transfer(address, stableTokenAmount).sendAndWaitForReceipt(),
-    ])
+    const goldAmount = await convertToContractDecimals(argv.gold, goldToken)
+    const stableTokenAmount = await convertToContractDecimals(argv.dollars, stableToken)
+    console.log(
```

Comment:
nit: might be nice to break this into three so we can see if/when the {gold from account, gold from reserve, C$ from account} transfers are made